examples/awesome-sensor-logger/docker-compose.yml

```diff
@@ -17,3 +33,18 @@ services:
       STREAM: phone
     depends_on:
       - proton
+
+  grafana:
+    image: grafana/grafana:latest
+    pull_policy: always
+    ports:
+      - 3000:3000
+    environment:
+      GF_AUTH_ANONYMOUS_ENABLED: 1
+      GF_AUTH_ANONYMOUS_ORG_ROLE: Admin
+      GF_INSTALL_PLUGINS: timeplus-proton-datasource
+    volumes:
+      - ./grafana_provisioning:/etc/grafana/provisioning
+    depends_on:
+      init-proton:
+        condition: service_completed_successfully
```

Comment:
You may like to put another empty line after

Reply:
I tested all docker compose files. Without the empty line in the bottom, it can work, but I am not sure whether having the empty last line is a best practice or not. Maybe nice to have

Reply:
Saw some discussions https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file  More like to avoid unexpected errors for old tools.